### ISSUE TITLE
setup.py --hpy-abi=universal build_ext --inplace doesn't work

### ISSUE THREAD

**Issue by antocuni** [hpyproject/hpy#140]
As the title says:

```
$ cd proof-of-concept
$ rm build -rf
$ python3 setup.py --hpy-abi=universal build_ext --inplace
...
writing hpy universal stub loader for pofpackage.foo to build/lib
copying build/lib/pof.hpy.so -> 
writing hpy universal stub loader for pof to build/lib
error: build/lib/pof.py already exists! Please delete.
```

cc @hodgestar

**Comment by antocuni** [comment]
I also tried to comment out the "please delete" check in `hpy/devel/__init__.py`, but things still don't work:

```diff
diff --git a/hpy/devel/__init__.py b/hpy/devel/__init__.py
index 33be1a0..c232029 100644
--- a/hpy/devel/__init__.py
+++ b/hpy/devel/__init__.py
@@ -240,8 +240,8 @@ class build_hpy_ext_mixin:
             ext._full_name, output_dir)
         stub_file = (os.path.join(output_dir, *ext._full_name.split('.')) +
                      '.py')
-        if compile and os.path.exists(stub_file):
-            raise DistutilsError(stub_file + " already exists! Please delete.")
+        ## if compile and os.path.exists(stub_file):
+        ##     raise DistutilsError(stub_file + " already exists! Please delete.")
         ext_file = os.path.basename(ext._file_name)
         module_name = ext_file.split(".")[0]
         if not self.dry_run:
```

```
$ python3 setup.py --hpy-abi=universal build_ext --inplace
...
copying build/lib/pof.hpy.so -> 
writing hpy universal stub loader for pof to build/lib
copying build/lib/pofpackage/foo.hpy.so -> pofpackage
writing hpy universal stub loader for pofpackage.foo to build/lib

$ python3 -c 'import pof'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'pof'
```

It seems that the stub `pof.py` is not copied to the inplace directory. If I copy it manually, it works.

**Comment by hodgestar** [comment]
Fixed in #143.
